magento2/docker-compose.yml

```diff
@@ -0,0 +1,99 @@
+version: '3.3'
+
+services:
+  ################################################
+  ####          MariaDB Container          ######
+  ##############################################
+  db_m2:
+    image: nosto/mariadb:10.0
```

Comment:
Why do we use a Nosto one and not https://hub.docker.com/layers/mariadb/library/mariadb/10.5.5/images/sha256-a7803e8939d15e80ac32eaea4b072884b3d335133fb679c7cd103fad75dafa9c?context=explore

Reply:
`I have increased inno_db values to get rid of some warning when the indexer
runs with default configuration`